CapabilityStatement/CapabilityStatement-nhsdigital.json

```diff
@@ -1,13 +1,14 @@
 {
     "resourceType": "CapabilityStatement",
-    "id": "nhsdigital-requirements",
-    "url": "https://fhir.nhs.uk/CapabilityStatement/nhsdigital-requirements",
+    "id": "England-CapabilityStatement-IGrequirements",
```

Comment:
Is the case correct for this? England-CapabilityStatement-IGrequirements, should it be England-CapabilityStatement-IGRequirements?  Also is IGRequirements a useful part of the name here.  Would NHSEngland-CapabilityStatement as an id and name not make more sense?  Same for instances below